mm2src/coins/lightning.rs

```diff
@@ -741,6 +785,74 @@ impl SwapOps for LightningCoin {
         }
     }
 
+    fn can_be_released(&self) -> bool { true }
```

Comment:
In my opinion, `can_be_released` is not clear enough. Could you please give it another name?
At least, `can_release_payment` or `can_fail_htlc_backwards`

Reply:
`can_be_released` is removed after implementing this suggestion https://github.com/KomodoPlatform/atomicDEX-API/pull/1592#discussion_r1063375403 by @artemii235